rivanna/README.md

```diff
@@ -0,0 +1,183 @@
+# Running Cylon on Rivanna
+
+Arup Sarker (arupcsedu@gmail.com, djy8hg@virginia.edu)
+
+## Issues on Summit
+
+Running cylon on rivanna requires a special configuration. We have verified
+that installations based on conda with virtual environment
+
+The solution is to use the guide we provide here.
+
+## Intsall instructions
+
+Clone the repository
+
+```shell
+git clone https://github.com/cylondata/cylon.git
+cd cylon
+```
+
+There are two ways you can build cylon on Rivanna
+
+### Fix all dependencies for Cylon on Rivanna without the dependencies of the system gcc and openmpi version.
+
+
+
+1. Change in yml for conda build: conda/environments/cylon.yml
+Add this libraries
+  - gcc
+  - gxx
+  - gxx_linux-64
+
+
+```shell
+name: cylon_rivanna
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.10
+  - cmake>=3.23.1,!=3.25.0
+  - arrow-cpp=9
+  - pyarrow=9.0.0
+  - glog
+  - openmpi=4.1.3=ha1ae619_105
+  - ucx>=1.12.1
+  - cython>=0.29,<0.30
+  - numpy
+  - pandas>=1.0,<1.6.0dev0
+  - fsspec>=0.6.0
+  - setuptools
+  # they are not needed for using pygcylon or compiling it
+  - pytest
+  - pytest-mpi
+  - mpi4py
+  - gcc
+  - gxx
+  - gxx_linux-64
+  

```

Comment:
@arupcsedu just a suggestion, would it be nice to keep this in a `requirements.txt` and just mention the usage in the `README.md`. This way, we can keep minimal edits to the markdown file but add dependencies to the `requirements.txt`.

Reply:
And if users has additional dependencies, they can create `requirements_extras.txt` to include their own dependencies but use our `requirements.txt` as the base dependencies.

Reply:
I think its better to not have a requirements_extra.txt but a 
requirements-rivanna.txt
The reason is that on other platforms you may have different versions of non python libraries as default, and you want to make sure that default software provided by the rivanna system staff can be used by having an explicit rivanna requirements.txt.
Also as this is conda, should it not be in something like
conda/environmenst/rivanna/cylon.yaml
?

Reply:
@vibhatha, This is the readme file and I tried to make minimal changes to the existing default build yaml file.
This are the additional default packages:   
- gcc
- gxx
- gxx_linux-64
are needed on top of existing packages.

It is not ensured that these packages will be the same after 2 years.  During that time, there can build errors. 
Moreover, I have found another way to make Cylon compatible with Rivanna. So, developers have to maintain it and might get confused.  We tried to make it as simple as possible. That's why after discussing with @nirandaperera , we have decided to make an additional readme file.

Reply:
@laszewsk , we thought to create a different yaml file for Rivanna. But we have two solutions and both solution needs minimal different changes on the default cylon yaml file. So, we kept both options open for developers.